include/mbedtls/ssl.h

```diff
@@ -1384,6 +1384,40 @@ struct mbedtls_ssl_config
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3)
     /** Allowed TLS 1.3 key exchange modes.                                 */
     int MBEDTLS_PRIVATE(tls13_kex_modes);
+
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+    /**
+     *  Server: The maximum amount of 0-RTT data that the client is allowed to
+     *          send when using this ticket, in bytes.
+     *          - 0      Disable early_data.
+     *          - others The maximum amount.
+     *  Client: Enable/disable early_data
+     *          - 0      Disable early_data
+     *          - others enable early_data
+     */
+    size_t MBEDTLS_PRIVATE(max_early_data_size);
```

Comment:
To make the code more readable, I don't prefer to re-use the field, if we want to benifit from the code size, suggest adding the defined(MBEDTLS_SSL_SRV_C) or defined(MBEDTLS_SSL_CLI_C) on them.